shared/chat/conversation/input-area/normal/platform-input.native.tsx

```diff
@@ -221,9 +234,19 @@ type ActionProps = {
 }
 
 const Action = React.memo(
-  ({hasText, insertMentionMarker, isEditing, onSubmit, openFilePicker, openMoreMenu}: ActionProps) => {
+  ({
```

Comment:
lets not explode these things in the param list. can pass as `props` and just pull it out inside the function